src/containers/catalog/CatalogActionFactories.js

```diff
@@ -19,4 +19,11 @@ export function createEntitySetListFailure(errorMessage: string) {
     type: actionTypes.ENTITY_SET_LIST_FAILURE,
     errorMessage
   }
+}
+
+export function createUpdateFilters(filterParams: {keyword: string, propertyTypeIds: string[], entityTypeId: string}) {
```

Comment:
that's a lot of Flow typing for a single parameter, and it makes it difficult to read. I'd suggest either doing `createUpdateFilters(filterParams :Object)`, or declaring a type somewhere, and referencing that instead.